.github/workflows/python-lint.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: black --check python/hsfs
```

Comment:
We can run black --check on the entire python dir as well

Reply:
good point! on it.